src/main/kotlin/org/rust/ide/presentation/Utils.kt

```diff
@@ -76,6 +90,7 @@ private fun presentableName(psi: RsElement): String? {
                 append(typeParameterBounds(psi))
             }
         }
+        is RsLetDecl -> (psi.pat as? RsPatIdentImpl)?.patBinding?.name
```

Comment:
Do you really need `RsPatIdentImpl` here and below (`RsStructureViewModel.kt`), or maybe just `RsPatIdent`?

Reply:
Yes, `RsPatIdent` is enough